### PR TITLE
Add mha142's file

### PR DESCRIPTION
**Pull request by mha142**
branch > file > commit > pull request
Done!

### CODE REVIEW

**Comment by drmahaha** [comment]
good.

**Comment by drmahaha** [comment]
good!

File: _posts/0000-01-02-mha142.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (drmahaha) [inline _posts/0000-01-02-mha142.md:5]: Dr. Maha

**Review by mha142 — COMMENTED** [review]
Done!

**Pull request closed without merging**